src/test/java/StepDefinitions/CourseWorkSteps.java

```diff
@@ -18,7 +18,6 @@
 
 public class CourseWorkSteps extends BaseTest {
 
-
     @Test
```

Comment:
@Test annotation is not needed here and in other methods
Gerkin type annotations usage is enough